server/jvm/build.gradle.kts

```diff
@@ -14,10 +14,10 @@ subprojects  {
     dependencies {
         implementation(platform("global.genesis:genesis-bom:${properties["genesisVersion"]}"))
         implementation("org.agrona:agrona:1.10.0!!")
```

Comment:
should probably get rid of this as well

Reply:
Yes, I have noted that and similar we need to do for position-app-seed